packages/params/test/e2e/overridePreset.test.ts

```diff
@@ -21,8 +21,7 @@ describe("Override preset", function () {
   vi.setConfig({testTimeout: 30_000});
 
   it("Should correctly override preset", async () => {
-    // These commands can not run with minimal preset
-    if (process.env.LODESTAR_PRESET === "minimal") delete process.env.LODESTAR_PRESET;
+    if (process.env.LODESTAR_PRESET) delete process.env.LODESTAR_PRESET;
```

Comment:
what's the reason for also deleting `LODESTAR_PRESET=mainnet` now?

Reply:
> We need to delete this env anyway because we override preset as hardcoded

I think the reason we have to delete it is this assertion
https://github.com/ChainSafe/lodestar/blob/c98a48a4fbbd2a4fd970d327683bf62161a93344/packages/params/test/e2e/overridePresetOk.ts#L20

The previous comment `// These commands can not run with minimal preset` does not make sense, any preset value fails this.

we should properly comment here why the preset env is removed

```
// `LODESTAR_PRESET` must not be set to correctly test if override works as expected
```